CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
+## v0.3.0 Venice
+
+The parser and semantic analysis stage of Solang have been through
+[a security audit](https://github.com/solana-labs/security-audits/blob/master/solang/Trail_of_Bits_Solang_Final_report.pdf). All security issues have been fixed.
+
+### Added
+- The CLI now has a `--release` option which disables printing of errors [salaheldinsoliman](https://github.com/salaheldinsoliman)
+- **Substrate**: chain extensions can be now used.
+  [xermicus](https://github.com/xermicus)
+
+### Fixed
+- Solidity error definitions are now parsed.
+  [seanyoung](https://github.com/seanyoung)
+- The Ethereum Solidity parser and semantic analysis tests are now run on Solang sema during
+  `cargo test`.
+  [seanyoung](https://github.com/seanyoung)
+- If a function returned a `storage` reference, then not returning a value explicitly is an error, since
+  the reference must post refer to an existing variable.
```

Comment:
Post refer?

Reply:
I think that was a brain misfire, I am sure I was thinking about the variables post function execution. Anyway fixed